### PR TITLE
Feedback permissions

### LINKED ISSUE #1170: Regular users are able to edit others API feedback
Steps: 
1. Visit http://nightly.apinf.io  
2. Login as a regular user-1 by clicking on 'Sign In'  
3. Go to 'Menu' and click on 'Catalog'   
4. Select and click on any API backend   
5. Go to 'Feedback  and click on 'Send Feedback' button   
6. Enter 'Topic' of the feedback report  
7. Enter 'Message'   
8. Choose 'message type' from drop down menu (eg. Feedback)  
9. Click on 'Submit' button   
10. Sign out   
11. Sign in as regular user-2   
12. Go to 'Menu' and click on 'Catalog'     
13. Select and click on same API backend as in above step 4  
14. Go to 'Feedback    
15. Click on 'Remove' button   

Expected Result: Regular users should not be allowed to edit/remove others API feedback. They are allowed to edit only their own feedback.

Actual: Regular users are able to edit/remove others API feedback.

### LINKED ISSUE #1744: Api tabs: anonymous user has access to feedback manipulation
Verified on nightly
Anonymous user has access to feadback manipulation

**Steps**:

1. Create any feedback in api
1. Sign out
1. Go to this api and open Feedback tab

**Found result**:
joxi_screenshot_1476109134917

**Expected result**: Remove feedback must be available only to api owner.

**OS**: debian 8
**Browser**: Chrome 52

### PR DESCRIPTION
**Pull request by jykae**
Closes #1744 
Closes #1170 

- template permission check
- DB allow rules for update/remove feedback

### CODE REVIEW

File: feedback/client/single/single.js [hunk 1]
```diff
@@ -43,15 +53,29 @@ Template.singleFeedback.helpers({
...
+    // Get single API Backend
+    const api = Apis.findOne(apiId);
+
+    return api;
   },
 });
 
 Template.singleFeedback.events({
-  'click .up-vote': function (event, template) {
+  'click .up-vote': () => {
```
Comment (marla-singer) [inline feedback/client/single/single.js]: I observed You corrected code which is associated with vote system. I went ahead and merged yours two branch for feedback fixing. Arrow function broken all vote system. Please come back to previous code version.

File: feedback/client/single/single.js [hunk 2]
```diff
@@ -61,7 +85,7 @@ Template.singleFeedback.events({
     // Submit upvote (+1) for current feedback
     Meteor.call('submitVote', feedbackId, 1);
   },
-  'click .down-vote': function (event, template) {
+  'click .down-vote': () => {
```
Comment (marla-singer) [inline feedback/client/single/single.js]: Please come back to previous code version.

File: feedback/client/single/single.js (hunk 1 above)
Comment (jykae) [inline feedback/client/single/single.js]: @marla-singer can you provide more info or screenshot, how does it break? voting was broken and part of other PR.

Comment (marla-singer) [inline feedback/client/single/single.js]: @jykae Meteor console error `Exception while invoking method 'submitVote' Error: Feedback id is required`. 
How to reproduce:
1. ` git checkout bugfix/feedback-voting `
2. Repeat these changes for `feedback/client/single/single.js`
3. Go to any api, open Feedback tab and try to vote

Possible solution: doesn't use an arrow function

**Comment by jykae** [comment]
@marla-singer changed arrow functions back to previous.

**Comment by marla-singer** [comment]
@jykae Merge it

**Pull request merged**